addons/kubeflow-pipeline/tools/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04
+
+# common environemnt variables
+ENV SHELL /bin/bash
```

Comment:
Can we move tools/  and jpg file out?  I'm not sure these files truly need put in this addon repo.

Reply:
That's fine. They are important for users and we don't have other place to put them.
We don't need to be obsessed with such details for now.